eth/ethconfig/config.go

```diff
@@ -141,6 +142,9 @@ type Config struct {
 	// Enables tracking of SHA3 preimages in the VM
 	EnablePreimageRecording bool
 
+	// Enables VM tracing
+	VMTracer vm.EVMLogger
```

Comment:
This is weird. This config object is decoded from the yaml config files. Shouldn't put objects like this here.